### PR TITLE
feat(@desktop/keycard): sign transaction using `Authenticate` flow

### PR DESCRIPTION
**Pull request by saledjenic**
Actually this is not a signing transaction, but rather authenticating logged in user when he wants to send a transaction. An authentication is done by entering password(regular user) or pin(keycard user).

A real signing transaction feature will be (hopefully) added in a near future where we're going to sign a transaction on a keycard which corresponds to a certain account, a user wants to send a transaction from.

To sum up... this change just removes password from the send modal and introduces `Authenticate` flow instead.

Fixes: #7510

### CODE REVIEW

File: src/app/modules/main/wallet_section/transactions/module.nim [hunk 1]
```diff
@@ -10,12 +10,26 @@ import ../../../../../app_service/service/network/service as network_service
 
 export io_interface
 
+type TmpTransactionDetails = object
```
Comment (alaibe) [inline src/app/modules/main/wallet_section/transactions/module.nim:13]: nit: SendTransactionDetails, this is the info we use to create the transaction and not the transaction by himself

**Review by alaibe — APPROVED** [review]

File: src/app/modules/main/wallet_section/transactions/module.nim (hunk 1 above)
Comment (saledjenic) [reply src/app/modules/main/wallet_section/transactions/module.nim]: Fixed, also added a short note that type should not be public ever, it's introduced like a local type to that module, just to bound and keep things simpler.

**Review by MishkaRogachev — APPROVED** [review]
Everything looks good for me

**Review by glitchminer — APPROVED** [review]
Tested, approved.
